DEPENDENCIES.md

```diff
@@ -4,17 +4,17 @@
 | ------------------------- | ---------------- | -------- | ------------------------------------- |
 | autoconf                  | >= 2.69          |    All   |                                    No |
 | boost                     | >= 1.65.0        |    All   |                                    No |
-| clang                     | >= 8.0           |    All   |                 Yes, if gcc installed |
-| clang-format              | >= 8.0           |    All   |                      Yes (formatting) |
-| clang-tidy                | >= 8.0           |    All   |                         Yes (linting) |
+| clang                     | >= 9.0           |    All   |                 Yes, if gcc installed |
+| clang-format              | >= 9.0           |    All   |                      Yes (formatting) |
+| clang-tidy                | >= 9.0           |    All   |                         Yes (linting) |
 | cmake                     | >= 3.9           |    All   |                                    No |
 | gcc                       | >= 9.1           |    All   | Yes, if clang installed, not for OS X |
 | gcovr                     | >= 3.2           |    All   |                        Yes (coverage) |
 | graphviz                  | any              |    All   |             Yes (query visualization) |
 | libnuma-dev               | any              |    Linux |                            Yes (numa) |
 | libnuma1                  | any              |    Linux |                            Yes (numa) |
 | libpq-dev                 | >= 9             |    All   |                                    No |
-| llvm-8-tools              | 8                |    Linux |                                    No |
+| llvm-9-tools              | 9                |    Linux |                                    No |
```

Comment:
Do you know why these are needed, is it for coverage?

Reply:
Should they also be part of the doggäfile?

Reply:
```
llvm-9-tools is already the newest version (1:9-2).
llvm-9-tools set to manually installed.
```

Will remove it

Reply:
```
llvm-9-tools is already the newest version (1:9-2).
llvm-9-tools set to manually installed.
```

As I did not add it to the Dockerfile, it appears as if this is part of a different package. Will remove it.